version/version.go

```diff
@@ -19,6 +19,7 @@ import (
 	"runtime"
 )
 
+// Deprecated: please use autorest.Version() instead.
 // Number contains the semantic version of this SDK.
 const Number = "v11.2.7"
```

Comment:
maybe it makes to sense to keep incrementing this version until we release the modules